front/components/assistant/conversation/lib.ts

```diff
@@ -160,6 +161,13 @@ export async function submitMessage({
   );
 
   if (!mRes.ok) {
+    if (mRes.status === 413) {
+      return new Err({
+        type: "content_too_large",
+        title: "Your message could not be sent.",
+        message: "Please try again with a shorter message",
```

Comment:
NIT, feel free to ignore:  
- Missing a dot to end the description. 
- I'd put the error directly in the title to be super clear, maybe: "Your message is too long to be sent."